### ISSUE TITLE
VOC dataset question

### ISSUE THREAD

**Issue by rudylyh** [Jia-Research-Lab/PFENet#27]
Hello! Thanks for sharing your code. I noticed that your setting on the VOC dataset is different from the PANet (https://github.com/kaixin96/PANet). Their training data and test data are from the training set. Your training data are 15 classes from the training set and test data are 5 classes from the validation set. The numbers of images in each class are also different. May I ask why there are two different settings? Are the performances on two settings comparable? Thank you in advance.

**Comment by tianzhuotao** [comment]
@rudylyh  Thanks for your interest in our work!

We follow the descriptions in OSLSM to build our dataset where the images for validation should be collected from the validation set of Pascal VOC. You can refer to the paper of [OSLSM](https://arxiv.org/abs/1709.03410) for more details (at the end of section 5 in OSLSM).

_"We follow a similar procedure to
form the test set D_test, but here the image-label pairs are taken from PASCAL VOC validation set and the corresponding label-set is L_test"._

Also, we follow the instruction on [SBD](http://home.bharathh.info/pubs/codes/SBD/download.html):

_"Please note that the train and val splits included with this dataset are different from the splits in the PASCAL VOC dataset. In particular some "train" images might be part of VOC2012 val"._ 

Therefore we use the collection that excludes all images in the validation set as the training set.

We have tried to reproduce PANet on our dataset and the results are comparable.

**Comment by rudylyh** [comment]
Thanks for your detailed answer. It helps me a lot.
